great_expectations_cloud/agent/models.py

```diff
@@ -10,7 +10,7 @@
 
 class AgentBaseModel(BaseModel):  # type: ignore[misc] # BaseSettings is has Any type
     class Config:
-        extra: str = Extra.forbid
+        extra: str = Extra.allow
```

Comment:
Can we change this to just ignore extra fields?
```suggestion
        extra: str = Extra.ignore
```
Going from `forbid` to `allow` seems drastic.

If we start `allowing` extra fields the schema loses a lot of its value.